bitnami/solr/templates/ingress.yaml

```diff
@@ -48,7 +50,11 @@ spec:
     {{- if .Values.ingress.tls }}
     - hosts:
         - {{ .Values.ingress.hostname }}
+      {{- if .Values.ingress.existingSecretName }}
+      secretName: {{ .Values.ingress.existingSecretName }}
+      {{- else }}
       secretName: {{ printf "%s-tls" .Values.ingress.hostname }}
```

Comment:
Shouldn't we also be disabling this secret if specifying the existing secret name option?

Reply:
nice, I missed that. updated